unit_tests/Test_Main.cpp

```diff
@@ -49,11 +49,20 @@
 #include <Kokkos_RemoteSpaces.hpp>
 
 int main(int argc, char *argv[]) {
+  int mpi_thread_level_available;
+  int mpi_thread_level_required = MPI_THREAD_SINGLE;
+
+  if (!std::is_same<Kokkos::Serial, Kokkos::DefaultHostExecutionSpace>::value)
```

Comment:
This doesn't work if `Kokkos::Serial` is not defined (which is why the tests are failing).

Reply:
Fixed.